topology/mask.h

```diff
@@ -388,15 +393,22 @@ class EllipseMask : public Mask< D >
   Position< D > center_;
   double major_axis_;
   double minor_axis_;
-  double intermediate_axis_;
-  double angle_;
+  double polar_axis_;
+  double azimuth_angle_;
+  double polar_angle_;
+
+  double x_scale_;
+  double y_scale_;
+  double z_scale_;
 
-  double x_dividend_;
-  double y_dividend_;
-  double z_dividend_;
+  double azimuth_cos_value_;
+  double azimuth_sin_value_;
 
-  double cosine_value_;
-  double sine_value_;
+  /*
+   * @returns boundary box of ellipse/ellipsoid.
+   */
+  void create_bbox();
```

Comment:
- This does not return the bbox, it sets it; the comment needs to be adapted.
- Since it is a private member function, its name should end with an `_`
- It is confusing to mix methods and data members; it would be better to move this declaration before the data member declarations.